frameworks/hdfs/src/main/dist/hdfs_svc.yml

```diff
@@ -166,24 +179,14 @@ plans:
         strategy: serial
         pod: name
         steps:
-          - pod-instance: 0
-            tasks: [format]
-          - pod-instance: 0
-            tasks: [node]
-          - pod-instance: 1
-            tasks: [bootstrap]
-          - pod-instance: 1
-            tasks: [node]
+          0: [format, node]
```

Comment:
Sorry, I'm actually not too clear on what the indices here are doing. Is it a pod-instance index, or a step index?